README.md

```diff
@@ -113,16 +125,17 @@ icon and then `Publish Data`.
 
 Fill in the form for publishing and submit. 
 For simplicity, you can enter any name for group, artifact and version.
-Use the URI of the file we prepared for publishing (`https://raw.githubusercontent.com/dbpedia/databus/68f976e29e2db15472f1b664a6fd5807b88d1370/README.md`) 
+Use the URI of the file we prepared for publishing [here](https://raw.githubusercontent.com/dbpedia/databus/68f976e29e2db15472f1b664a6fd5807b88d1370/README.md) 
```

Comment:
This I would keep as it was before, it is supposed to be used for copy/paste and not for clicking on it

Reply:
Okayy got it